common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -87,7 +87,9 @@
     private static final boolean DIRECT_BUFFER_PREFERRED =
             HAS_UNSAFE && !SystemPropertyUtil.getBoolean("io.netty.noPreferDirect", false);
     private static final long MAX_DIRECT_MEMORY = maxDirectMemory0();
-    private static final int MAX_MPSC_CAPACITY =  1024 * 1024; // TODO: Maybe make this configurable ?
+    private static final int MPSC_CHUNK_SIZE =  1024;
```

Comment:
I would keep it like it is for now... If we need we can add it later

> Am 23.06.2016 um 16:27 schrieb Nathan Mittler <notifications@github.com>:
> 
> In common/src/main/java/io/netty/util/internal/PlatformDependent.java:
> 
> > @@ -87,7 +87,9 @@
> >      private static final boolean DIRECT_BUFFER_PREFERRED =
> >              HAS_UNSAFE && !SystemPropertyUtil.getBoolean("io.netty.noPreferDirect", false);
> >      private static final long MAX_DIRECT_MEMORY = maxDirectMemory0();
> > -    private static final int MAX_MPSC_CAPACITY =  1024 * 1024; // TODO: Maybe make this configurable ?
> > +    private static final int MPSC_CHUNK_SIZE =  1024;
> Should you make chunk size configurable while you're at it?
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>